ArduPlane/quadplane.cpp

```diff
@@ -3603,10 +3618,14 @@ float QuadPlane::forward_throttle_pct()
     // integrator as throttle percentage (-100 to 100)
     vel_forward.integrator += fwd_vel_error * deltat * vel_forward.gain * 100;
 
-    // inhibit reverse throttle and allow petrol engines with min > 0
-    int8_t fwd_throttle_min = plane.have_reverse_thrust() ? 0 : plane.aparm.throttle_min;
+    // inhibit to minimum throttle unless fwd_thr_min is set
+    int8_t fwd_throttle_min;
+    if (fwd_thr_min < 0){
+        fwd_throttle_min = fwd_thr_min;
+    } else{
```

Comment:
```suggestion
    } else {
```

Reply:
By initialising to zero we could combine the existing ternary into the if. 
```suggestion
    } else if (!plane.have_reverse_thrust()) {
        fwd_throttle_min = plane.aparm.throttle_min;
```